src/CMakeLists.txt

```diff
@@ -99,6 +99,19 @@ if(${TRITON_ENABLE_S3})
   message(STATUS "Using aws-sdk-cpp ${AWSSDK_VERSION}")
 endif()
 
+#
+# OpenTelemetry
+#
+if (NOT WIN32)
+  if(${TRITON_ENABLE_TRACING})
+    find_package(absl CONFIG REQUIRED)
+    find_package(CURL CONFIG REQUIRED)
+    find_package(nlohmann_json CONFIG REQUIRED)
+    find_package(opentelemetry-cpp CONFIG REQUIRED)
```

Comment:
Do we have to run find_package on each of the opentelemetry dependency individually? Should opentelemetry find its dependency internally?

Reply:
For some reason, build fails without them. Opentelemetry in these past day released a new version 1.9.0, when I try it later, I'll check if we can remove them